### PR TITLE
modified the invite box

### PR DESCRIPTION
**Pull request by SparshKat**
Fixed the issue #1229 . Although the only thing i had trouble with was small font for 'new code for every 72 hours'. Still review this one, if styling issues still seem prominent let me know.

### CODE REVIEW

**Comment by SparshKat** [comment]
Yes i purposefully left the styling parts, as i needed to consult you upon this. Is there any styling sheet that has been globally used for hubs where i can easily style the above things, or shall i do the in-line styling. In-line styling is not a good thing to do in a huge project so i needed to ask this beforehand

**Comment by gfodor** [comment]
yes there are `scss` files in the repo in `src/assets/stylesheets` where we do all of our styling

**Comment by SparshKat** [comment]
@gfodor  Does this look better now? 
Screenshot 2020-01-12 at 12 12 23 AM

**Comment by SparshKat** [comment]
@gfodor Could you review this PR now, or suggest if something more needs to be added ? :)

**Comment by gfodor** [comment]
Sorry I missed this! We will take a look this week. Thanks for the PR!

File: src/assets/stylesheets/invite-dialog.scss [hunk 1]
```diff
@@ -119,6 +119,15 @@
   }
 }
 
+:local(.linkcode){
```
Comment (gfodor) [inline src/assets/stylesheets/invite-dialog.scss]: Please adjust this to use the `link-code` naming convention, and then use `linkCode` to reference the styles in the JS file. thanks!

Comment (SparshKat) [reply src/assets/stylesheets/invite-dialog.scss]: Yes, i'm sorry i almost forgot the naming conventions. But i've fixed the naming conventions ! Hope this works :)

Comment (SparshKat) [reply src/assets/stylesheets/invite-dialog.scss]: And i'd love to contribute more do tag me in issues if you want me to :)

**Comment by gfodor** [comment]
This looks good! I will push it. Thank you for the contribution!

**Comment by SparshKat** [comment]
Yes Love to contribute here! please do merge so that i can pull the changes later on! :)

**Comment by gfodor** [comment]
Sorry I forgot to push this last week! will do so now.

**Pull request merged**